rllib/algorithms/impala/impala_learner.py

```diff
@@ -136,6 +152,49 @@ def rl_module_required_apis(cls) -> list[type]:
 ImpalaLearner = IMPALALearner
 
 
+class _GPULoaderThread(threading.Thread):
+    def __init__(
+        self,
+        *,
```

Comment:
Use the name argument here and pass it to super

Reply:
done